examples/capi.c

```diff
@@ -142,14 +142,12 @@ static void get_block_hash(struct evmc_uint256be* result,
 }
 
 /// EVM log callback.
-///
-/// @note The `evm_log` name is used to avoid conflict with `log()` C function.
-static void evm_log(struct evmc_context* context,
-                    const struct evmc_address* address,
-                    const uint8_t* data,
-                    size_t data_size,
-                    const struct evmc_uint256be topics[],
-                    size_t topics_count)
+static void emit_log(struct evmc_context* context,
```

Comment:
Ack this change.

Reply:
The ABI is bumped in this PR already, but that's not needed in this case, because this is a static function in a example.

Reply:
Right I thought it was renaming the member, but that was renamed before.